sopel/trigger.py

```diff
@@ -106,30 +110,40 @@ class PreTrigger:
     component_regex = re.compile(r'([^!]*)!?([^@]*)@?(.*)')
     intent_regex = re.compile('\x01(\\S+) ?(.*)\x01')
 
-    def __init__(self, own_nick, line, url_schemes=None):
+    def __init__(
+        self,
+        own_nick: tools.Identifier,
+        line: str,
+        url_schemes: Optional[Sequence] = None,
+        identifier_factory: IdentifierFactory = tools.Identifier,
+    ):
```

Comment:
`tools.Identifier` still? Is this a proof that the move hasn't affected backward compatibility? 😂

Reply:
That's the goal here, yes!

Reply:
But truly I should change to `identifiers.Identifier`, would be better.